src/NHibernate.Test/Async/Insertordering/FamilyModel/Fixture.cs

```diff
@@ -104,8 +105,8 @@ public async System.Threading.Tasks.Task CircularReferencesAsync()
 			using (var session = OpenSession())
 			using (var tran = session.BeginTransaction())
 			{
-				Assert.AreEqual(9, await (session.Query<Woman>().CountAsync()));
-				Assert.AreEqual(6, await (session.Query<Man>().CountAsync()));
+				Assert.AreEqual(9, session.Query<Woman>().Count());
+				Assert.AreEqual(6, session.Query<Man>().Count());
```

Comment:
@maca88 do you know why it did not pick up the `CountAsync` here?

Reply:
The problem lies in this part of the `NHibernate.Test.csproj`:
```xml
<ItemGroup Condition="'$(TargetFramework)'=='net6.0'">
    <Compile Remove="**\CfgTest\DefaultNsAssmFixture.cs" />
    <Compile Remove="**\CfgTest\HbmOrderingFixture.cs" />
    <Compile Remove="**\DynamicProxyTests\PeVerifyFixture.cs" />
    <Compile Remove="**\Insertordering\**" />
    <Compile Remove="**\Linq\ODataTests.cs" />
    <Compile Remove="**\NHSpecificTest\NH1850\**" />
    <Compile Remove="**\NHSpecificTest\NH2484\**" />
    <Compile Remove="**\NHSpecificTest\NH2188\**" />
    <Compile Remove="**\NHSpecificTest\NH3121\**" />
  </ItemGroup>
```

When the generator loads the project by using `net6.0` as the target framework, files matching the above globs won't be included and therefore the code won't be generated for them. By deleting the whole `Async` folder and re-generating the async code again, this fixture along with some others matching these globs won't be generated. They are not removed by the generator, because it removes only generated files that are present in the loaded project prior code generation.

Reply:
Thanks